### ISSUE TITLE
Feature: Allow users to mark decks as favorite.

### ISSUE THREAD

**Issue by kzcheng** [PennyDreadfulMTG/Penny-Dreadful-Tools#7631]

**Comment by bakert** [comment]
Theory: if you really like a deck you will play it. Then it'll be on your player page for all time. So this feature may not be that useful. Maybe it tracks the decks you are planning on playing?

**Comment by silasary** [comment]
This comes back to the idea of a "Register in League with this deck" button.

**Comment by kzcheng** [comment]
Yeah, I was thinking about a way to add a few decks as favorite, so I can play them in future league runs.

As in, I'm currently in a league run, and I saw a nice deck I want to steal. So I want to be able to mark it as favorite, and copy it for myself later.

It also might be useful when I want to study a deck. For example, I study Briar Moss's decks every season, so I find it helpful to cherry pick some versions with interesting changes, and be able to compare them when deciding my own deck list.

But yeah, obviously all of those requirements can be fulfilled with just browser bookmarks, but still, could be a nice feature.

**Comment by kzcheng** [comment]
Another thing is, when I saw multiple decks I'm interested in, I don't want to be forced to register multiple 0 game runs in the league. This can feel a bit too counter intuitive.

But I do like the idea of a "Register in League with this deck" button. As in users can now register a deck first before finding cards and building it.

**Comment by kzcheng** [comment]
An example of the deck favorite feature can be found on [NetrunnerDB](https://netrunnerdb.com/).

It's a deckbuilder website for a different game. It's probably my favorite deckbuilder / card game database I have ever seen, but it's not for Magic, and I don't play that game anymore, and that game is also discontinued.

My dream is being able to write a deckbuilder just like that for magic, and attach it to the PD website or something. But it do seems very challenging.

**Comment by bakert** [comment]
netrunnerdb is pretty slick. It's open source – https://github.com/Alsciende/netrunnerdb/ :)

**Comment by kzcheng** [comment]
Oh cool, looks like it's written in PHP.

**Comment by bakert** [comment]
"enter a league with this deck" requested again by Cabooseman today. Maybe it is a good idea?
